Add integrations page (#461)
cloudflare/workers-docs: content/tooling/integrations.md

---
title: 'Integrations'
new: true
---

> We're currently working on building integrations for Cloudflare Workers with other tools, especially if they make continuous integration and continuous deployment easier. Below is a list of the current integrations officially supported by Cloudflare Workers:

Reviewer: ```suggestion
We're currently working on building integrations for Cloudflare Workers to optimize your continuous integration and continuous deployment flow. The integrations officially supported for Cloudflare Workers are:
```
